azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/implementation/BatchAccountsInner.java

```diff
@@ -44,7 +45,7 @@
  * An instance of this class provides access to all the operations defined
  * in BatchAccounts.
  */
-public final class BatchAccountsInner {
+public final class BatchAccountsInner implements SupportsAsyncListing<Page<BatchAccountInner>> {
```

Comment:
Manual change in the Inner class, will be autogenerated in case we choose in approach.